src/js/views/patients/patient/patient.scss

```diff
@@ -57,7 +57,7 @@
 .patient__tab--selected {
   display: inline-block;
   font-size: 24px;
-  line-height: 1;
+  line-height: 0.9;
   margin-right: 4px;
   padding: 0 16px 16px;
 }
```

Comment:
I think cleaner than this would be to modify `patient__context-trail`
We can make the padding `16px 0;` and then add something like
```css
// NOTE: Margin added to account for sidebar icon.
margin-bottom: 6px;
```

Reply:
Hmm ya, that would work!